### ISSUE TITLE
1917. Leetcodify Friends Recommendations

### ISSUE THREAD

**Issue by AlexndrGetcel** [LeetCode-Feedback/LeetCode-Feedback#9171]
getcel

- [ ] Question
Hello, accurate solution is not accepted by the test system due to time limit.

I am pretty sure that I have done the minimum number of algorithms' steps in order to get a correct answer.
In addition, already existent correct decisions are not supposed to be accepted too.

T-sql

#### Code you used for Submit/Run operation
``` 
select user_id, recommended_id
from (
    select distinct dr.user_id, dr1.user_id as recommended_id
    from   Listens dr
          ,Listens dr1 
    where dr.user_id != dr1.user_id 
          and dr.song_id = dr1.song_id
          and dr.day = dr1.day
    group by dr.user_id, dr1.user_id, dr.day 
    having(count(distinct(dr.song_id))) >= 3 
) o
where not exists(
select null
from (
    select user1_id, user2_id from Friendship
    union all
    select user2_id, user1_id from Friendship 
) i where i.user1_id = user_id and recommended_id = user2_id)
```

**Comment by AlexndrGetcel** [comment]
Hello, I have been using T- sql, but the code was written the way compatible with all platforms. Thanks in advance,Alex G 23.09.2022, 14:56, "LC_M" ***@***.***>: Thank you for reaching out to us. To help us further assess this issue and reproduce this issue on our end, would you please share the following information with us:The language you submitted to reach this result.We appreciate your time and support. If you have any other questions, please don't hesitate to let us know.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by LC-Pam** [comment]
Thank you for your time. The code received Accepted, https://leetcode.com/submissions/detail/806833976/
We appreciate your support!

**Comment by AlexndrGetcel** [comment]
But it hasn't  been accepted from my work station   23.09.2022, 15:29, "LC_M" ***@***.***>: Thank you for your time. The code received Accepted, https://leetcode.com/submissions/detail/806833976/We appreciate your support!—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by AlexndrGetcel** [comment]
Could you please label my solution as resolved Thanks 23.09.2022, 15:38, "al g" ***@***.***>:But it hasn't  been accepted from my work station   23.09.2022, 15:29, "LC_M" ***@***.***>: Thank you for your time. The code received Accepted, https://leetcode.com/submissions/detail/806833976/We appreciate your support!—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by AlexndrGetcel** [comment]
Hello, Could you please mark the task as completed in my profile? I am still not capable of accepting my accurate answer. Thanks is advance! 23.09.2022, 14:56, "LC_M" ***@***.***>: Thank you for reaching out to us. To help us further assess this issue and reproduce this issue on our end, would you please share the following information with us:The language you submitted to reach this result.We appreciate your time and support. If you have any other questions, please don't hesitate to let us know.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
